src/network.cpp

```diff
@@ -169,7 +170,9 @@ namespace net {
   addr_to_url_escaped_string(boost::asio::ip::address address) {
     address = normalize_address(address);
     if (address.is_v6()) {
-      return "["s + address.to_string() + ']';
+      std::stringstream ss;
```

Comment:
This change to me seems unnecessary? Why was this needed? Some reallocations might be avoided, but is it worth it?

Reply:
I seem to recall that under C++20, the return type of `boost::asio::ip::address::to_string()` changes which makes "summing" these illegal using the current syntax. I can't find the compilation error in my CI history though.

I might have been able to fix it by adding parentheses, but we use stringstreams elsewhere in sunshine (see another patch in this change), so I figured this would be fine.